src/index.ts

```diff
@@ -5,3 +5,7 @@ export * from './parsers/oval';
 export * from './parsers/xccdf';
 export * from './utilities/diff';
 export * from './utilities/update';
+import Control from './objects/control';
+export {Control};
```

Comment:
Does this do something different than `export * from './objects/control';`? I know we did have issues before with trying to access the control type, but it is interesting if we have to do it explicitly.